modes/reftex/evil-collection-reftex.el

```diff
@@ -119,23 +138,35 @@
 
   (evil-set-initial-state 'reftex-toc-mode 'normal)
 
-  ;; This one is more involved, in reftex-toc.el, line 282 it shows the prompt
+  ;; This one is more involved, in reftex-toc.el, line 267 it shows the prompt
   ;; string with the keybinds and I don't see any way of changing it to show evil-like binds.
   (evil-collection-define-key 'normal 'reftex-toc-mode-map
     "j" 'reftex-toc-next
     "k" 'reftex-toc-previous
+    "go" 'reftex-toc-view-line
+    "gO" 'reftex-toc-view-line
     (kbd "RET") 'reftex-toc-goto-line-and-hide
     (kbd "<tab>") 'reftex-toc-goto-line
     "g?" 'reftex-toc-show-help
+    "?" 'reftex-toc-show-help
     "q" 'reftex-toc-quit
-    "ZZ" 'reftex-toc-quit
+    "ZZ" 'reftex-toc-quit-and-kill
     "ZQ" 'evil-quit
     "gr" 'reftex-toc-rescan
-    "r" 'reftex-toc-rescan
+    "gR" 'reftex-toc-Rescan
     "l" 'reftex-toc-toggle-labels
-    "?" 'reftex-toc-show-help
+    "i" 'reftex-toc-toggle-index
+    "c" 'reftex-toc-toggle-context
+    "F" 'reftex-toc-toggle-file-boundary
+    "rl" 'reftex-toc-rename-label
+    "zi" 'reftex-toc-display-index
     "x" 'reftex-toc-external
-    ;; (kbd "SPC") 'reftex-toc-view-line
+    "." 'reftex-toc-show-calling-point
+    "gs" 'reftex-toc-jump
```

Comment:
Maybe "J", I believe we usually use that for jump.

Reply:
Makes sense. Fixed that!